bundlewrap/items/pkg_openbsd.py

```diff
@@ -40,6 +65,7 @@ class OpenBSDPkg(Item):
     BUNDLE_ATTRIBUTE_NAME = "pkg_openbsd"
     ITEM_ATTRIBUTES = {
         'installed': True,
+        'flavor': "",
```

Comment:
Shouldn't flavor default to `None`?

>flavor = None,   version = "-foo":  "pkgname--foo"  (backwards compat)

Reply:
I think `flavor` should to default to `""`, since I believe that this is what you want most of the time.

It will be a change in behavior, yes. You might call it a "fix", because bw previously simply failed to install a package -- unless you set `version` to `-foo`, which is neither obvious nor documented.

If you prefer to have it default to `None`, I'd like to add a TODO item to change it to `""` for bw 4.0.